kitchen-sink-demo/src/app/demos/leaky-faucet/components/leaky-faucet.component.ts

```diff
@@ -0,0 +1,63 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'leaky-faucet',
+  template: `
+  <div>
+  <p>{{ status }}</p>
+  <button type="button" class="btn btn-danger"
+    (click)="toggleLeak()">
```

Comment:
In this case, I would probably keep this (click) on the previous line. Unless your going to add a new line for every attribute.